src/components/application/style.ts

```diff
@@ -0,0 +1,96 @@
+import styled from "styled-components";
+import Button from "../button";
+import { colors } from "src/constants";
+
+
+export const Container = styled.div`
+  color: ${colors.white};
+  flex-direction: column;
+  background-color: ${colors.alto};
+`;
+
+export const H1 = styled.h1`
+  align-items: center;
+  font-weight: 700;
+  font-size: 60px;
+  line-height: 74px;
+  text-transform: uppercase;
+  width: 940px;
```

Comment:
Do not add big width for H1 block, this fixed size maybe only for head containers